solar/cell.py

```diff
@@ -0,0 +1,21 @@
+from solar import SolarDay
+
+class SolarCell:
+    def __init__(self, area, temperature):
+        self.area = area
+        # efficiency of solar energy conversion to electrical energy
+        self.efficiency = .237 
+        # temp_coeff is the percentage decrease
+        # in the cells' efficiency for every degree above 25C
+        self.temp_coefficient = -.0029
+        # the internal temperature of the cells/array
+        self.temperature = temperature
+        
+    def total_efficiency_decrease(self):
+        if self.temperature > 25:
```

Comment:
What happens if the temperature is less than 25?

Reply:
Added comments to explain.